From 275da9f4ae3d34ac1d507d0e3e38da8d6ee856c0 Mon Sep 17 00:00:00 2001
From: YD Tseng <Yd_Tseng@asmedia.com.tw>
Date: Thu, 17 Mar 2016 11:35:57 +0800
Subject: [PATCH 07/10] gpio: amdpt: Add a new ACPI HID

This patch adds a new ACPI HID, AMDIF030, in the pt_gpio_acpi_match.

Signed-off-by: YD Tseng<Yd_Tseng@asmedia.com.tw>
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
Signed-off-by: Sudheesh Mavila <sudheesh.mavila@amd.com>
---
 drivers/gpio/gpio-amdpt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpio/gpio-amdpt.c b/drivers/gpio/gpio-amdpt.c
index cbbb966..afe9d7a 100644
--- a/drivers/gpio/gpio-amdpt.c
+++ b/drivers/gpio/gpio-amdpt.c
@@ -241,6 +241,7 @@ static int pt_gpio_remove(struct platform_device *pdev)
 
 static const struct acpi_device_id pt_gpio_acpi_match[] = {
 	{ "AMDF030", 0 },
+	{ "AMDIF030", 0 },
 	{ },
 };
 MODULE_DEVICE_TABLE(acpi, pt_gpio_acpi_match);
-- 
2.7.4

